frontend/apps/filemanager/filemanagercollection.lua

```diff
@@ -0,0 +1,124 @@
+local ButtonDialogTitle = require("ui/widget/buttondialogtitle")
+local FileManagerBookInfo = require("apps/filemanager/filemanagerbookinfo")
+local InputContainer = require("ui/widget/container/inputcontainer")
+local Menu = require("ui/widget/menu")
+local UIManager = require("ui/uimanager")
+local Screen = require("device").screen
+local _ = require("gettext")
+
+local FileManagerCollection = InputContainer:extend{
+    coll_menu_title = _("Favorites"),
+}
+
+function FileManagerCollection:init()
+    self.ui.menu:registerToMainMenu(self)
+end
+
+function FileManagerCollection:addToMainMenu(menu_items)
+    -- insert table to main tab of filemanager menu
```

Comment:
Well, technically just wherever MenuSorter is told to put it by the user. ;-)

But this is the same as every addToMainMenu function; I don't think a comment adds much?

```suggestion
```